Makefile

```diff
@@ -1,19 +1,51 @@
 DOT := $(shell command -v dot 2> /dev/null)
 
-up:
+rm-up:
 	docker network inspect ssdcrmdockerdev_default >/dev/null || docker network create ssdcrmdockerdev_default
 	docker compose -f dev.yml -f rm-services.yml up -d ${SERVICE} ;
 	pipenv install --dev
 	./setup_pubsub.sh
+
+rh-up:
+	docker network inspect ssdcrmdockerdev_default >/dev/null || docker network create ssdcrmdockerdev_default
+	docker compose -f rh-dev.yml -f rh-services.yml up -d ${SERVICE} ;
+	pipenv install --dev
+	./setup_pubsub.sh
+
+rmrh-up:
+	docker network inspect ssdcrmdockerdev_default >/dev/null || docker network create ssdcrmdockerdev_default
+	docker compose -f dev.yml -f rm-services.yml -f rh-dev.yml -f rh-services.yml up -d ${SERVICE} ;
+	pipenv install --dev
+	./setup_pubsub.sh
 	
-down:
+rm-down:
+	@echo "IF RH IS RUNNING, THIS WILL STOP PUBUSB EMULATOR FOR THOSE SERVICES TOO"
```

Comment:
Can we solve this by keeping the common services in a separate YAML and not stopping them on these specific `down` commands?